package-lock.json

```diff
@@ -5,7 +5,7 @@
     "requires": true,
     "packages": {
         "": {
-            "version": "1.26.0-SNAPSHOT",
+            "version": "1.28.0-SNAPSHOT",
```

Comment:
Necessary?

Reply:
Yeah our release scripts aren't updating the version in the package lock it looks like